pr2eus/robot-interface.l

```diff
@@ -441,6 +456,7 @@
 - min-time : minimum time for time to goal
 - end-coords-interpolation : set t if you want to move robot in cartesian space interpolation
 - end-coords-interpolation-steps : number of divisions when interpolating end-coords
+- limit-rotation : set t if you want to restrict upper/lower limit angle of unlimited rotational joint
```

Comment:
Please add this doc string to :angle-vector, too

Reply:
Thank you for your review.
I fixed it.